x-pack/plugins/features/server/oss_features.ts

```diff
@@ -468,3 +483,118 @@ const timelionFeature: KibanaFeatureConfig = {
     },
   },
 };
+
+const reportingFeatures: {
+  [Property in
+    | 'generateCsvDiscover'
+    | 'downloadCsvDashboard'
+    | 'generateScreenshotDashboard'
+    | 'generateScreenshotVisualize']: SubFeatureConfig;
+} = {
+  generateCsvDiscover: {
+    name: i18n.translate('xpack.features.ossFeatures.reporting.discoverGenerateCSVFeatureName', {
+      defaultMessage: 'CSV Reports',
+    }),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'generate_csv_report',
+            name: i18n.translate('xpack.features.ossFeatures.reporting.discoverGenerateCSV', {
+              defaultMessage: 'Generate CSV reports',
+            }),
+            includeIn: 'all',
+            savedObject: { all: [], read: [] },
+            management: { insightsAndAlerting: ['reporting'] },
+            api: ['generateReport'],
+            ui: ['generateCsv'],
+          },
+        ],
+      },
+    ],
+  },
+  downloadCsvDashboard: {
+    name: i18n.translate('xpack.features.ossFeatures.reporting.dashboardDownloadCSVFeatureName', {
+      defaultMessage: 'Download Saved Search CSV',
+    }),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'download_csv_dashboard_panel',
+            name: i18n.translate('xpack.features.ossFeatures.reporting.dashboardDownloadCSV', {
+              defaultMessage: 'Download CSV reports from saved searches',
+            }),
+            includeIn: 'all',
+            savedObject: { all: [], read: [] },
+            management: { insightsAndAlerting: ['reporting'] },
+            api: ['downloadCsv'],
+            ui: ['downloadCsv'],
+          },
+        ],
+      },
+    ],
+  },
+  generateScreenshotDashboard: {
+    name: i18n.translate(
+      'xpack.features.ossFeatures.reporting.dashboardGenerateScreenshotFeatureName',
+      {
+        defaultMessage: 'PDF / PNG Reports',
+      }
+    ),
+    privilegeGroups: [
+      {
+        groupType: 'independent',
+        privileges: [
+          {
+            id: 'generate_screenshot_report',
+            name: i18n.translate(
+              'xpack.features.ossFeatures.reporting.dashboardGenerateScreenshot',
+              {
+                defaultMessage: 'Generate PDF or PNG reports',
+              }
+            ),
+            includeIn: 'all',
+            minimumLicense: 'platinum' as 'platinum',
+            savedObject: { all: [], read: [] },
+            management: { insightsAndAlerting: ['reporting'] },
+            api: ['generateReport'],
+            ui: ['generateScreenshot'],
+          },
+        ],
+      },
+    ],
+  },
+  generateScreenshotVisualize: {
+    name: i18n.translate(
+      'xpack.features.ossFeatures.reporting.visualizeGenerateScreenshotFeatureName',
+      {
+        defaultMessage: 'PDF / PNG Reports',
```

Comment:
opinion: I'd just call this `Reporting`. If we add other report types, they can be grouped within this bucket in the future. Same comment applies for the other privilege groups here